tests/foreman/cli/test_contenthost.py

```diff
@@ -567,115 +559,38 @@ def setUp(self):
 
```

Comment:
Do we need to run the setUp()  separately before test_contenthost_get_errata_info() and test_contenthost_apply_errata()? Is it not enough to run setUp() once in this class? I am trying to save time from running redundant setup steps.